app/javascript/dashboard/components/ChatList.vue

```diff
@@ -119,6 +121,11 @@ export default {
     this.$store.dispatch('setChatFilter', this.activeStatus);
     this.resetAndFetchData();
     this.$store.dispatch('agents/get');
+
+    bus.$on('openNextChat', (chatId) => {
+      const element = this.$refs['conversation-card-' + chatId][0]
```

Comment:
Hey can we use `es6` string literals here ?
```
`conversation-card-${chatId}`
```

Also how about we split this before accessing 0th index? Something like this ( feel free to change the logic )

```
const chatRef = this.$refs[`conversation-card-${ chatId}`];
const [element = {}]= `chatRef;`
const notClickable = !element. cardClick;

if (notClickable) return;
```

Reply:
62d4695 + https://github.com/chatwoot/chatwoot/pull/757/commits/ce29edebad36fb559e8aca26f435095738444e50